src/study_api.jl

```diff
@@ -1,4 +1,75 @@
-const PSRCLASSES_DEFAULTS = JSON.parsefile(joinpath(@__DIR__, "JSON", "psrclasses.defaults.json"))
+const PSRCLASSES_DEFAULTS = JSON.parsefile(joinpath(@__DIR__, "json_metadata", "psrclasses.defaults.json"))
+const DATE_FORMAT_1 = Dates.DateFormat(raw"yyyy-mm-dd")
+const DATE_FORMAT_2 = Dates.DateFormat(raw"dd/mm/yyyy")
+
+function list_attributes(
+    data::Data,
+    name::String,
+)
+    if !haskey(data.data_struct, name)
+        error("PSR Class '$name' is not available for this study")
+    end
+
+    class_struct = data.data_struct[name]
+
+    attrs = sort(collect(keys(class_struct)))
+
+    return attrs
+end
+
+function list_attributes(
+    data::Data,
+    name::String,
+    index::Integer,
```

Comment:
This a really a minor comment but `Integer` is actually an abstract type, usually, it is better to define in terms of the most concrete types possible, in this case, `Int` which is an alias for `Int64` should be better. I know that there are `Integer`s in other parts but we should try to narrow to the concrete type in those functions too.